Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,329 @@
+                if (isCurrent)
+                {
+                    isCurrentIndexShift = balances.Length - 1;
+                }
+                for (int j = 0 + isCurrentIndexShift; j < balances.Length; j++)
+                {
+                    if(balances[j] == balance)
+                    {
+                        isBalance = true;
+                    }
+                }
+                if (isBalance)
+                {
+                    totalNames += $"{workingPeopleAndBalances[i][0]}, ";
+                }
+            }
+            string[] formatedNames = FormatBalanceNames(totalNames);
+            return formatedNames;
+        }
+
+        /// <summary>
+        /// Takes a string of names separated by ', ' and returns a grammatically correct string
+        /// </summary>
+        /// <param name="balanceNames"></param>
+        /// <returns></returns>
+        public static string[] FormatBalanceNames(string balanceNames)
+        {
+            int count = balanceNames.Split(',').Length - 1;
+            balanceNames = balanceNames.Remove(balanceNames.Length - 2);
+            string[] formatedBalanceNames = { "", "is", "person", "have" };
+            if (count == 1)
+            {
+                formatedBalanceNames[0] = balanceNames;
+                return formatedBalanceNames;
+            }
+            string[] names = balanceNames.Split(',');
+            StringBuilder sbNames = new StringBuilder(names[0]);
+            if (count == 2)
+            {
+                sbNames.Append($" and{names[1]}");
+                formatedBalanceNames[0] = sbNames.ToString();
+            }
+            else
+            {
+                for (int i = 1; i < names.Length - 1; i++)
+                {
+                    sbNames.Append($",{names[i]}");
+                }
+            }
+            
+            sbNames.Append($" and{names[^1]}");
+            formatedBalanceNames[0] = sbNames.ToString();
+            formatedBalanceNames[1] = "are";
+            formatedBalanceNames[2] = "people";
+            formatedBalanceNames[3] = "has";
+            return formatedBalanceNames;
+        }
+
+        /// <summary>
+        /// Takes one individual with balances as an array and return their greatest loss in history.
+        /// </summary>
+        /// <param name="personAndBalance"></param>
+        /// <returns></returns>
+        public static float GetBiggestLoss(string[] personAndBalance)
+        {
+            float[] balance = ConvertStringArrToFloatArr(personAndBalance[1..]);
+            float loss = 0f;
+            for (int i = 0; i < balance.Length - 1; i++)
+            {
+                if (balance[i + 1] - balance[i] < loss)
+                {
+                    loss = balance[i + 1] - balance[i];
+                }
+            }
+            return loss;
+        }
+
+        public static string ConcatMoney(float balance)
+        {
+            if (balance < 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5156603)